hoomd/md/pair/pair.py

```diff
@@ -110,6 +110,13 @@ class Pair(force.Force):
 
         Type: `TypeParameter` [`tuple` [``particle_type``, ``particle_type``],
         `float`])
+
+    .. py:attribute:: mode
+
+        *mode*, *optional*: defaults to ``none``.
+        Possible values: ``none``, ``shift``, ``xplor``
```

Comment:
String literal values should be wrapped in quotes (unless this conflicts with other HOOMD conventions). If this change is accepted, please also update the formatting in the docstring above these lines.
```suggestion
        *mode*, *optional*: defaults to ``"none"``.
        Possible values: ``"none"``, ``"shift"``, ``"xplor"``
```